### PR TITLE
Add deficiency diseases

### PR DESCRIPTION
**Pull request by vikashvverma**
Adde deficiency disease section and updated existing sections.

IA Page: [https://duck.co/ia/view/vitamin_cheat_sheet](https://duck.co/ia/view/vitamin_cheat_sheet)

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Vitamin Cheat Sheet](https://duck.co/ia/view/vitamin_cheat_sheet)

**Description**: It displays the sources and benefits of each vitamin

**Example Query**: [Vitamin Cheat Sheet](https://beta.duckduckgo.com/?q=Vitamin Cheat Sheet), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**:  

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/vitamin_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/vitamin_cheat_sheet)*

File: share/goodie/cheat_sheets/json/vitamin.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "template_type": "reference",
     "section_order": [
         "Source",
-        "Benefits"
+        "Benefits",
+        "Deficiency Disease"
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/vitamin.json]: maybe `Symptoms of Deficiency` ?

Comment (vikashvverma) [inline share/goodie/cheat_sheets/json/vitamin.json]: @mintsoft Symptoms are initial phase of disease.

But the section contains disease and not symptoms.

I think it's okay.

What do you say?

**Comment by MrChrisW** [comment]
@vikashvverma Thanks for the contribution. 

Unfortunately the information contained in the `Deficiency Disease` section is not broad enough and only applies to extreme deficiencies or can be caused primarily by other factors. Providing this information for the vitamin query isn't helpful for users. I'm going to close this PR. 

In future we may want to include a more general "hypovitaminosis" symptoms. :+1:

**Pull request closed without merging**